### ISSUE TITLE
⛔🐞 Bug Report: Lazy Load Background Images experiment Breaks Twitter card Images on Twitter

### ISSUE THREAD

**Issue by foxmahoney** [elementor/elementor#20640]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Haven't found anything related after extensive google search. It took awhile to figure out exactly which feature was breaking the twitter cards.

### Steps to reproduce

Turning on Lazy Load Background Images in Experiments then checking my Twitter account. The links to my site that should display Twitter cards (large image) simply don't (excerpts are shown but no image whatsoever). Turning the experiment back off, the twitter cards work again.

I have Autoptimize and WP Fastest Cache on my site with the Elementor install as well as Cloudflare. At first I thought these were the culprits until I was able to isolate the problem to exactly the Lazy Load feature.

About the system info: I have no problems (so far) with the plugins listed as incompatible and I've been using them quite a long time. I hope these items don't diminish the validity of my observation.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.3.37
	PHP Version: 7.4.33
	PHP Memory Limit: 768M
	PHP Max Input Vars: 5000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://fcpxtemplates.com
	Home URL: https://fcpxtemplates.com
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 256M
	Max Memory limit: 384M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: America/Chicago
	Admin Email: sightcreations@gmail.com
	Debug Mode: Inactive

== Theme ==
	Name: freestore child
	Version: 1.2.1
	Author: Kaira
	Child Theme: Yes
	Parent Theme Name: FreeStore
	Parent Theme Version: 1.2.2
	Parent Theme Author: Kaira

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:107.0) Gecko/20100101 Firefox/107.0

== Active Plugins ==
	Anywhere Elementor
		Version: 1.2.6
		Author: WPVibes

	Autoptimize
		Version: 3.1.4
		Author: Frank Goossens (futtta)

	Checkout Field Editor for WooCommerce
		Version: 1.8.0
		Author: ThemeHigh

	Child Theme Configurator
		Version: 2.6.0
		Author: Lilaea Media

	Classic Editor
		Version: 1.6.2
		Author: WordPress Contributors

	Code Snippets
		Version: 3.2.2
		Author: Code Snippets Pro

	Custom Fonts
		Version: 1.3.7
		Author: Brainstorm Force

	Donations via PayPal
		Version: 1.9.9
		Author: Tips and Tricks HQ, Johan Steen

	Dynamic Visibility for Elementor
		Version: 5.0.2
		Author: Dynamic.ooo

	Elementor
		Version: 3.9.0
		Author: Elementor.com

	Elementor Header & Footer Builder
		Version: 1.6.13
		Author: Brainstorm Force, Nikhil Chavan

	Elements Plus!
		Version: 2.15
		Author: The CSSIgniter Team

	Essential Addons for Elementor
		Version: 5.5.0
		Author: WPDeveloper

	Exclusive Addons Elementor
		Version: 2.6.1
		Author: DevsCred.com

	Footer Putter
		Version: 1.17
		Author: Russell Jamieson

	Fullwidth Page Templates
		Version: 1.1.1
		Author: Brainstorm Force

	Health Check & Troubleshooting
		Version: 1.5.1
		Author: The WordPress.org community

	Hide Page And Post Title
		Version: 1.5.6
		Author: Arjun Thakur

	Login No Captcha reCAPTCHA (Google)
		Version: 1.7
		Author: Robert Peake and Contributors

	LuckyWP Table of Contents
		Version: 2.1.4
		Author: LuckyWP

	Media Library Organizer
		Version: 1.5.0
		Author: WP Media Library

	OMGF
		Version: 5.5.0
		Author: Daan from Daan.dev

	Post Type Switcher
		Version: 3.2.1
		Author: Triple J Software, Inc.

	Premium Addons for Elementor
		Version: 4.9.41
		Author: Leap13

	Product Customer List for WooCommerce
		Version: 3.1.2
		Author: Kokomo

	Rank Math SEO
		Version: 1.0.103.1
		Author: Rank Math

	Really Simple SSL
		Version: 6.0.12
		Author: Really Simple Plugins

	Remove Category Base
		Version: 1.1.0
		Author: LittleBizzy

	Shortcodes Finder
		Version: 1.5.1
		Author: Scribit

	Simple Download Monitor
		Version: 3.9.15
		Author: Tips and Tricks HQ, Ruhul Amin, Josh Lobe

	SlickNav Mobile Menu
		Version: 1.9.1
		Author: Neil Gee

	StoreCustomizer
		Version: 2.4.5
		Author: Kaira

	UpdraftPlus - Backup/Restore
		Version: 1.22.23
		Author: UpdraftPlus.Com, DavidAnderson

	Widget Detector for Elementor
		Version: 1.2.5
		Author: ElementInvader & FreelancersTools (Ivica Delić)

	WishSuite - Wishlist for WooCommerce
		Version: 1.3.2
		Author: HasTheme

	WooCommerce
		Version: 7.1.0
		Author: Automattic

	WooCommerce - Autocomplete Orders
		Version: 1.1.4
		Author: Serious Plugins

	Wordfence Assistant
		Version: 1.0.9
		Author: Wordfence

	Wordfence Security
		Version: 7.8.0
		Author: Wordfence

	WP-Sweep
		Version: 1.1.5
		Author: Lester 'GaMerZ' Chan

	WP Fastest Cache
		Version: 1.0.8
		Author: Emre Vona

	WPForms Lite
		Version: 1.7.8
		Author: WPForms

	WP HTML Mail - Email Template Designer
		Version: 3.3.5
		Author: Hannes Etzelstorfer // codemiq

	WP Mail SMTP
		Version: 3.6.1
		Author: WPForms

	WP phpMyAdmin
		Version: 5.2.0.7
		Author: Puvox.software

	WP Rollback
		Version: 1.7.2
		Author: GiveWP.com

	WP Statistics
		Version: 13.2.8
		Author: VeronaLabs

== Must-Use Plugins ==
	Health Check Troubleshooting Mode
		Version: 1.8.0
		Author: 

== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active
	Additional Custom Breakpoints: Inactive by default
	Import Export Website Kit: Active by default
	Hide native WordPress widgets from search results: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Landing Pages: Active by default
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active
	Lazy Load Background Images: Inactive by default
	Save as Default: Active by default

== Log ==
	
Log: showing 20 of 252022-10-30 10:29:41 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Start  
2022-10-30 10:29:41 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Finished 
2022-10-30 10:29:41 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.7.8',
  'to' => '3.8.0',
)]
2022-10-30 10:29:43 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.7.8',
  'to' => '3.8.0',
)]
2022-10-30 10:29:46 [info] elementor::elementor_updater Started 
2022-10-30 10:29:46 [info] Elementor/Upgrades - _on_each_version Start  
2022-10-30 10:29:46 [info] Elementor/Upgrades - _on_each_version Finished 
2022-10-30 10:29:46 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Start  
2022-10-30 10:29:46 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Finished 
2022-10-30 10:29:46 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.7.8',
  'to' => '3.8.0',
)]
2022-11-13 12:29:16 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.0',
  'to' => '3.8.1',
)]
2022-11-13 12:29:29 [info] elementor::elementor_updater Started 
2022-11-13 12:29:29 [info] Elementor/Upgrades - _on_each_version Start  
2022-11-13 12:29:29 [info] Elementor/Upgrades - _on_each_version Finished 
2022-11-13 12:29:29 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.0',
  'to' => '3.8.1',
)]
2022-11-13 12:29:31 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.0',
  'to' => '3.8.1',
)]
2022-12-06 16:17:54 [info] elementor::elementor_updater Started 
2022-12-06 16:17:54 [info] Elementor/Upgrades - _on_each_version Start  
2022-12-06 16:17:54 [info] Elementor/Upgrades - _on_each_version Finished 
2022-12-06 16:17:54 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.1',
  'to' => '3.9.0',
)]

JS: showing 13 of 13JS: 2022-10-07 05:40:18 [error X 2][https://fcpxtemplates.com/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:739] a is null 
JS: 2022-10-07 23:40:16 [error X 6][https://fcpxtemplates.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.7.8:3:767305] T.originalEvent.key is undefined 
JS: 2022-10-23 04:33:36 [error X 2][https://fcpxtemplates.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.7.8:3:571433] elementor.documents.getCurrent() is null 
JS: 2022-11-06 01:27:32 [error X 21][https://fcpxtemplates.com/wp-admin/load-scripts.php?c=0&loadchunk_0=jquery-core,jquery-migrate,underscore,backbone,regenerator-runtime,wp-polyfill,wp-hooks&ver=6.1:2:51816] a is undefined 
JS: 2022-11-09 02:47:44 [error X 3][https://fcpxtemplates.com/wp-content/plugins/elementor/assets/js/frontend-modules.min.js?ver=3.8.0:2:4823] e is undefined 
JS: 2022-11-09 03:51:20 [error X 7][https://fcpxtemplates.com/wp-content/plugins/elementor/assets/js/new-template.min.js?ver=3.8.0:2:2869] elementor_new_template_form_controls is not defined 
JS: 2022-11-11 06:04:41 [error X 2][https://fcpxtemplates.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.8.0:3:694254] T.getContainer is not a function 
JS: 2022-11-18 23:08:56 [error X 15][https://fcpxtemplates.com/wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.8.1:2:14029] An invalid argument supplied as multiple control value [array (
  'category' => 'InvalidElementData',
  'tag' => 'Element `undefined` got as `exad_modal_image_gallery` value. Expected array or object.',
)]
JS: 2022-11-19 05:25:10 [error X 5][https://fcpxtemplates.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.8.1:3:588259] this.container is undefined 
JS: 2022-11-21 04:57:39 [error X 6][https://fcpxtemplates.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.8.1:3:295930] C.ui.form is undefined 
JS: 2022-11-24 20:25:50 [error X 1][https://fcpxtemplates.com/wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.8.1:2:15608] r[a] is undefined 
JS: 2022-11-25 23:05:19 [error X 4][https://fcpxtemplates.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.8.1:3:769417] this.store.find(...) is undefined 
JS: 2022-12-02 07:55:55 [error X 1][https://fcpxtemplates.com/wp-content/plugins/dynamic-visibility-for-elementor/assets/js/editor-dynamic-visibility.js?ver=5.0.1:205:3] elementorFrontend.config.elements is undefined 

PHP: showing 4 of 4PHP: 2022-11-08 22:37:27 [notice X 2][/home/fcpxte5/public_html/wp-content/plugins/elementor/includes/widgets/menu-anchor.php::156] Undefined index: anchor [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-10 23:18:25 [notice X 8][/home/fcpxte5/public_html/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-13 12:33:31 [notice X 24693][/home/fcpxte5/public_html/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::304] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-11-20 16:58:37 [notice X 1][/home/fcpxte5/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

== Elementor - Compatibility Tag ==
	
	Anywhere Elementor: Incompatible
	Dynamic Visibility for Elementor: Incompatible
	Elementor Header & Footer Builder: Incompatible
	Essential Addons for Elementor: Incompatible
	Exclusive Addons Elementor: Incompatible
	Premium Addons for Elementor: Incompatible
	Widget Detector for Elementor: Incompatible

**Comment by nicholaszein** [comment]
Hello @foxmahoney,

thanks for submitting your issue! 🙏 

Can you please show us an example of the images not appearing?

Can you also confirm you have set the correct featured images and Open Graph data in Rank Math? Or are you letting Twitter choose which image it will show?

**Comment by nicholaszein** [comment]
Hey @foxmahoney!

Thanks for letting me know, for closing the issue, and for being so patient. I wasn't able to answer you sooner. 🙏

I'm glad you solved your issue by yourself. I did suspect the issue was cache related but wanted to rule out all other options before continuing. 💪

Caching is a necessary evil, unfortunately. But fear not, I'm always here to help! I hope everything is going well so far. A piece of advice, **<ins>always</ins>** clear the cache after any update. I've learned this the difficult way with WordPress. 🤷

Best regards, and a happy new year! 😉 🥂
